src/game-config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"events": {
+		"pool": [
+			"BoomerGregorEvent", "CuteGirlEvent", "FratPartyEvent"
+		],
+		"followUp": [
+			"LandingEvent", "PickFacultyEvent", "PickResidenceEvent"
+		],
+		"seasonal": [
+			[],
+			[],
+			[],
+			[],
+			[],
+			[],
+			[],
+			[],
+			[],
+			[],
+			[],
+			[]
+		]
+	}
+}
```

Comment:
hmm, i think maybe something like this would be better?

```
let seasonal = [];
for (n=0; n <12; n++) {
	this.seasonal.push([]);
}
```

Reply:
Yes, but you can't do that in json.

Reply:
oh, i see. totally overlooked. ok i guess you can ignore that then. i thought this .js